File: Python Code/Daily_Flash_Code_0/Week10/26Mar/57-DailyFlash_Solutions/26_Mar_Solution_Three/Python/prog4.py
num = 1
for i in range(7):
    if(i < 4):
        for j in range(i + 1):
            if(j == 0):
                print("H", end = "")
            if(j == 1):
                print("E", end = "")
            if(j == 2 or j == 3):
                print("L", end = "")
    else:
        for j in range(7 - i):
        
            if(j == 0):
                print("H", end = "")
            if(j == 1):
                print("E", end = "")
            if(j == 2 or j == 3):
                print("L", end = "")
    print()

File: Python Code/Daily_Flash_Code_0/Week10/26Mar/57-DailyFlash_Solutions/26_Mar_Solution_Three/Java/prog4.java
class Demo {
	public static void main(String[] args) {
		int num = 1;
		for(int i = 0; i < 7; i++) {
			if(i < 4) {
				for(int j = 0; j < i + 1; j++) {
					if(j == 0)
						System.out.print("H");
					if(j == 1)
						System.out.print("E");
					if(j == 2 || j == 3)
						System.out.print("L");
				}
			} else {
				for(int j = 0; j < 7 - i; j++) {
					
					if(j == 0)
						System.out.print("H");
					if(j == 1)
						System.out.print("E");
					if(j == 2 || j == 3)
						System.out.print("L");
				}
			}
			System.out.println();
		}
	}
}
